src/Components/AllArticles.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import axios from 'axios';
+import { Link } from "react-router-dom";
+
+const AllArticles = () => {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
+        axios
+            .get(`https://nc-news-july.herokuapp.com/api/articles`)
+            .then(({ data }) => {
+                console.log(data.articles);
+                setArticles(data.articles);
+            });
+    }, []);
+    return (
+        <main id="all_main">
+            <ul>
+                {articles.map((article) => {
+                    return (
+                        <Link to={`/items/${article.article_id}`}>
+                            {' '}
+                            <li key={article.article_id.item_id}>{article.title} </li>
```

Comment:
You should add more than just your article title. How about author or date for more info?